Packs/CommonScripts/Scripts/ParseHTMLIndicators/ParseHTMLIndicators.py

```diff
@@ -0,0 +1,74 @@
+import re
```

Comment:
please use functions, it's ready hard to follow the code this way.

Reply:
I didn't use functions due to the type of the script and flow.
Will do an enhancement to the script with functions.